joomla/joomla-cms
# Fix for Broken html in Date of Birth field in User Profile Form

Pull Request for Issue #28883, see that for code details.

### Summary of Changes

This fixes the Date of Birth field in User Profile Edit Form that generates a broken html.

The issue is to due the extra info about the date format being added with broken html around it.

### Previous result

The previous "hacky" HTML creates an extra control-group, and the actual "controls" is outside of it, so two problems for the price of one.

The goal was to put the info language string on a separate line but I don't think that is worth breaking the HTML structure of the form.

### New result

Now the info sits near the label and doesn't break the HTML of the control-group. It does occupy a bit more space (with the default layout) but at least the HTML is correct.

### Testing Instructions

Simply see the HTML of the edit form of a user profile page with a DOB field enabled.

### Documentation Changes Required

None
